lib/full-site-editing/edit-site-page.php

```diff
@@ -119,6 +148,7 @@ static function( $classes ) {
 		'styles'                               => gutenberg_get_editor_styles(),
 		'defaultTemplateTypes'                 => $indexed_template_types,
 		'defaultTemplatePartAreas'             => get_allowed_block_template_part_areas(),
+		'__experimentalHomeTemplate'           => gutenberg_edit_site_resolve_home_template(),
```

Comment:
Why do we want to mark it as `experimental`? Maybe we should mark it as `internal` instead?

Reply:
We don't have an `internal` prefix. Generally `__unstable` is for things we know we'll never make public.